HoneyDo.Web/ClientApp/package.json

```diff
@@ -35,8 +29,10 @@
     "test": "react-scripts test --env=jsdom",
     "test:ci": "CI=true react-scripts test",
     "eject": "react-scripts eject",
-    "styleguide": "styleguidist server",
-    "styleguide:build": "styleguidist build"
+    "lint": "eslint src"
+  },
+  "eslintConfig": {
```

Comment:
This was the missing config that our app needed to use the `eslint` that `react-scripts` has baked in.

Reply:
I think we can just move this into the `.eslintrc.json` file I deleted.